setup.py

```diff
@@ -71,6 +71,12 @@
             '2k_earth_daymap.jpg',
         ],
     },
+    download_url="https://pypi.python.org/pypi/pyvista",
```

Comment:
There is no documentation about download_url. So I referenced the [code of NumPy](
https://github.com/numpy/numpy/blob/e5fcb9d52c1b9d3c9caf067849ad1bba128c7e17/setup.py#L440-L445).

Reply:
[setuptools docs says](https://setuptools.pypa.io/en/latest/references/keywords.html)
> `project_urls`
>     An arbitrary map of URL names to hyperlinks, allowing more extensible documentation of where various resources can be found than the simple `url` and `download_url` options provide.

which sounds like `download_url` would be replaced by `project_urls`. Not sure it makes sense to have a "download" link from PyPI that links to the exact same page on PyPI. This is exactly what [NumPy's "download" link seems to do](https://pypi.org/project/numpy/).

Reply:
I see, that is why there was no documentation. It was mimicking NumPy's method, but it makes no sense to link to itself, so we will remove it.
```suggestion
```